icubam/backoffice/handlers/upload.py

```diff
@@ -29,9 +29,9 @@ def post(self) -> None:
 
     sync = synchronizer.CSVSynchronizer(self.db)
     sync_fns: Dict[str, Callable[..., int]] = {
-      'user': sync.sync_users_from_csv,
-      'icu': sync.sync_icus_from_csv,
-      'bedcounts': sync.sync_bedcounts_from_csv
+      'Users': sync.sync_users_from_csv,
```

Comment:
i don t like too much this dependence here. it might not have been optimal either, but this `objtype` is really used to show something to the user, be translated, etc. I would rather use an enum so as to make sure things are consistent and do not depend on variables meant to be printed/displayed.

Reply:
I switched to an enum, thanks.